database/handlers/parentOrg.go

```diff
@@ -180,3 +188,40 @@ func parseResultSetToSlice(rowsRs *sql.Rows) ([]database.ParentOrg, error) {
 
 	return orgs, nil
 }
+
+/*
+ * Function to update organization details by ID.
+ *
+ * Method for:  Client
+ * Args:        id (organization ID), newDetails (ParentOrg object with updated details)
+ * Returns:     org (updated ParentOrg object)
+ */
+ func (client Client) UpdateOrganizationByID(id string, newDetails database.ParentOrg) *database.ParentOrg {
+	updateStmt :=
+		`
+        UPDATE parentOrgs
+        SET name = $2, logo = $3, description = $4, skills = $5, term = $7, website = $8
+        WHERE id = $1
+        RETURNING *;
+        `
+
+	rowsRs, err := client.Query(updateStmt, id, newDetails.Name, newDetails.Logo, newDetails.Description, newDetails.Skills, newDetails.Term, newDetails.Website)
```

Comment:
You can't pass slices directly in `Query` function as then Go wouldn't know how to convert them to a `pg.Array`. Refer to other controllers to see how to work with slices and arrays in database, or check online reference.